requirements.txt

```diff
@@ -1,18 +1,995 @@
+    --hash=sha256:4f1d9991f5acc0ca119f9d443620b77f9d6b33703e51011c16baf57afb285fc6 \
+    --hash=sha256:08695f5cb7ed6e0531a20572697297273c47b8cae5a63ffc6d6ed5c201be6e44
+decorator==5.1.1; python_version >= "3.6" \
+    --hash=sha256:b8c3f85900b9dc423225913c5aace94729fe1fa9763b38939a95226f02d37186 \
+    --hash=sha256:637996211036b6385ef91435e4fae22989472f9d571faba8927ba8253acbc330
+docker-pycreds==0.4.0; python_version >= "3.6" \
+    --hash=sha256:6ce3270bcaf404cc4c3e27e4b6c70d3521deae82fb508767870fdbf772d584d4 \
+    --hash=sha256:7266112468627868005106ec19cd0d722702d2b7d5912a28e19b826c3d37af49
+einops==0.6.0; python_version >= "3.7" \
+    --hash=sha256:c7b187a5dc725f079860ec2d330c1820448948622d826273345a8dd8d5f695bd \
+    --hash=sha256:6f6c78739316a2e3ccbce8052310497e69da092935e4173f2e76ec4e3a336a35
+filelock==3.9.0; python_version >= "3.7" and python_full_version >= "3.7.0" \
+    --hash=sha256:f58d535af89bb9ad5cd4df046f741f8553a418c01a7856bf0d173bbc9f6bd16d \
+    --hash=sha256:7b319f24340b51f55a2bf7a12ac0755a9b03e718311dac567a0f4f7fabd2f5de
+ftfy==6.1.1; python_version >= "3.7" and python_version < "4" \
+    --hash=sha256:0ffd33fce16b54cccaec78d6ec73d95ad370e5df5a25255c8966a6147bd667ca \
+    --hash=sha256:bfc2019f84fcd851419152320a6375604a0f1459c281b5b199b2cd0d2e727f8f
+gitdb==4.0.10; python_version >= "3.7" \
+    --hash=sha256:c286cf298426064079ed96a9e4a9d39e7f3e9bf15ba60701e95f5492f28415c7 \
+    --hash=sha256:6eb990b69df4e15bad899ea868dc46572c3f75339735663b81de79b06f17eb9a
+gitpython==3.1.31; python_version >= "3.7" \
+    --hash=sha256:f04893614f6aa713a60cbbe1e6a97403ef633103cdd0ef5eb6efe0deb98dbe8d \
+    --hash=sha256:8ce3bcf69adfdf7c7d503e78fd3b1c492af782d58893b650adb2ac8912ddd573
+typing-extensions==4.5.0; python_version >= "3.7" and python_full_version >= "3.7.0" and python_version < "3.10" \
+    --hash=sha256:fb33085c39dd998ac16d1431ebc293a8b3eedd00fd4a32de0ff79002c19511b4 \
+    --hash=sha256:5cb5f4a79139d699607b3ef622a1dedafa84e115ab0024e0d9c044a9479ca7cb
+huggingface-hub==0.12.1; python_full_version >= "3.7.0" \
+    --hash=sha256:867586cc8543fe1bd43a219fedbea7d71690021ad80f0c46f35c4751069278d7 \
+    --hash=sha256:6f960f6246ef9c3446d0d6275e853485515682c350917fdaf2a59705f8b9ebb3
+idna==3.4; python_version >= "3.7" and python_version < "4" and python_full_version >= "3.7.0" \
+    --hash=sha256:90b77e79eaa3eba6de819a0c442c0b4ceefc341a7a2ab77d7562bf49f425c5c2 \
+    --hash=sha256:814f528e8dead7d329833b91c5faa87d60bf71824cd12a7530b5526063d02cb4
+imageio==2.25.1; python_version >= "3.7" \
+    --hash=sha256:5bce7f88eef7ee4e9aac798d3b218fea2e98cbbaa59a3e37b730a7aa5784eeac \
+    --hash=sha256:6021d42debd2187e9c781e494a49a30eba002fbac1eef43f491bbc731e7a6d2b
+importlib-metadata==6.0.0; python_version < "3.9" and python_version >= "3.7" \
```

Comment:
is this normal? why are there packages with a required python version < 3.9?

Reply:
It's because of jsonschema (required by jsonmerge, required by k-diffusion) 
![Screenshot_20230218-103713_Brave](https://user-images.githubusercontent.com/77058942/219853337-a02cd3d0-427f-43ec-acd7-7a8cf66ec748.jpg)
It's ignored by default if it doesn't match AFAIK